google-cloud-tools-plugin/google-cloud-core/resources/messages/CoreMessageBundle.properties

```diff
@@ -33,4 +33,6 @@ error.dialog.disable.plugin.restart=Changes in plugin configuration will take ef
 error.dialog.disable.plugin.norestart=After disabling the plugin please restart {0} to apply changes in plugin configuration.
 error.dialog.disable.plugin.title=Disable Plugin
 error.dialog.disable.plugin.action.disable=&Disable
-error.dialog.disable.plugin.action.disableAndRestart=Disable and &Restart
\ No newline at end of file
+error.dialog.disable.plugin.action.disableAndRestart=Disable and &Restart
+
+cloud.tools.group.menu.text=Google Cloud Tools
```

Comment:
I wonder if this needs to be named differently than the one in the other module. i.e. do the group names here need to be unique

Reply:
hmm I think this was mistakenly added  here.

Reply:
ah yeah, cause I was experimenting with pulling the action registration on the menu into the GCS plugin.xml but that was less straight forward than I expected.  I'm going to put this message back in GCT.

Reply:
fixed.